runtime/redirector/forwarders.m4

```diff
@@ -412,8 +412,12 @@ _IF([JAVA_SPEC_VERSION >= 20],
 _IF([JAVA_SPEC_VERSION >= 21],
 	[_X(JVM_IsForeignLinkerSupported, JNICALL, false, jboolean, void)])
 _IF([JAVA_SPEC_VERSION >= 21],
-	[_X(JVM_VirtualThreadMount, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide, jboolean firstMount)])
+	[_X(JVM_VirtualThreadMount, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide)])
 _IF([JAVA_SPEC_VERSION >= 21],
-	[_X(JVM_VirtualThreadUnmount, JNICALL, false, void, JNIEnv *env, jobject vthread, jboolean hide, jboolean lastUnmount)])
```

Comment:
`JVM_VirtualThreadMount` and `JVM_VirtualThreadIUnmount` are still neede for Java 20 (with the old signatures).

Reply:
These methods are introduced in JDK21. JDK20 uses the old methods JVM_VirtualThreadMountBegin|End and JVM_VirtualThreadUnmountBegin|End, which are included above. References:
- JDK20: https://github.com/ibmruntimes/openj9-openjdk-jdk20/blob/1e863069cbd09f7165b9834e64e3305a642c56e3/src/java.base/share/classes/java/lang/VirtualThread.java#L1066-L1076
- JDK21: https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/509718f021217fb1c1be512df54024e65b86b9fc/src/java.base/share/classes/java/lang/VirtualThread.java#L1093-L1099